### ISSUE TITLE
I need a way to define an absolute path to the proxyUrl

### ISSUE THREAD

**Issue by Silverdust-ZH** [galamai/AspNetCore.Rendertron#1]
Hi - first of all - thank you very much for creating this awesome Middleware for the .net framework!
I am using this middleware within a docker environment having several containers (one running the ASP .Net Core application and another running the rendertrone service) running a common virtual network. I have configured your middleware like so: app.UseRendertron(proxyUrl: "http://rendertron:3000/render/");

This results in your middleware sending the following request to the rendertrone service:
http://rendertron:3000/render/http://127.0.0.1:8080, which of course, the rendertrone service is unable to find resulting in this error message:

```
> rendertron@2.0.0 start /rendertron
> node build/rendertron.js

Listening on port 3000
Error: net::ERR_CONNECTION_REFUSED at http://127.0.0.1:8080/
    at navigate (/rendertron/node_modules/puppeteer/lib/Page.js:598:37)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
```

Rendertrone is trying to connect on itself on port 8080, which is completely wrong.

What I would need was something like this: 
app.UseRendertron(proxyUrl: "http://rendertron:3000/render/http://airborne-aviation:8080");
In other words, I need a possibility to tell the middleware exactly how to call the rendertron service

Best regards and many thanks so far! 
Samuel

**Comment by galamai** [comment]
Hi, the original idea of rendertron lies precisely in the availability of a real host for it, and the substitution violates this principle. In the simplest case, a solution with address redefinition will work. But substitution will create problems for CEO.
Case with a localhost address and containers known. You need to set the desired name for any external address of the container and everything will work. To do this, you need to make changes to the hosts file of your system.
